x-pack/plugin/esql/src/main/antlr/EsqlBaseLexer.g4

```diff
@@ -424,3 +425,68 @@ SETTING_WS
     : WS -> channel(HIDDEN)
     ;
 
+
+//
+// METRICS command
+//
+mode METRICS_MODE;
+METRICS_PIPE : PIPE -> type(PIPE), popMode;
+
+fragment METRICS_UNQUOTED_IDENTIFIER_PART
+    : ~[=`|,[\]/ \t\r\n]
+    | '/' ~[*/] // allow single / but not followed by another / or * which would start a comment
+    ;
+
+METRICS_UNQUOTED_IDENTIFIER
+    : METRICS_UNQUOTED_IDENTIFIER_PART+ -> popMode, pushMode(CLOSING_METRICS_MODE)
```

Comment:
why us a separate mode and not follow the STATS implementation for now? This will simplify your grammar and remove some subtleties around mode management.

`FROM_UNQUOTED_IDENTIFIER -> type(FROM_QUOTED_IDENTIFIER)`